recipe/meta.yaml

```diff
@@ -1,170 +1,254 @@
-{% set version = "2.0.1" %}
-{% set sha256 = "9c564ca440265c69400ef5fdd48bf15e28af5aa4bed84c95efaad960a6699998" %}
+{% set version = "2.1.0" %}
+# Set the RC number to build release candidates. Set to None otherwise
+{% set rc = None %}
+{% set sha256 = "631c71f7f7d6174952f35b5ed4a45ec115720a4ef3eb619678de5893af54f403" %}
+{% set build_number = 0 %}
 
 package:
-  name: pytorch
+  name: pytorch-select
   version: {{ version }}
 
 source:
+{% if rc != None %}
+  git_url: https://github.com/pytorch/pytorch.git
```

Comment:
Could we use `https://github.com/pytorch/pytorch/archive/refs/tags/v{{ version }}-rc{{ rc }}.tar.gz` to download the release candidates instead of using git?

Reply:
yeah, I tried that originally and then changed it to this method. My commit messages don't say why I did it, but I suspect that it's because the refs/tags tarball doesn't contain the submodules

Reply:
the official releases do, but the archive/refs/tags tarballs is just a github-provided feature and only contains the repo contents

Reply:
Yep. What I was saying is that it's annoying that GH's default archives don't include submodules.